docs/imviz/plugins.rst

```diff
@@ -379,7 +379,12 @@ catalog dropdown menu.
     and works best when you only have a single image loaded in a viewer.
 
 To load a catalog from a supported `JWST ECSV catalog file <https://jwst-pipeline.readthedocs.io/en/latest/jwst/source_catalog/main.html#output-products>`_, choose "From File...".
-The file must be able to be parsed by `astropy.table.Table.read` and contain a column labeled 'sky_centroid'.
+The file must be able to be parsed by `astropy.table.Table.read` and contains the following columns:
+
+* ``'sky_centroid'``: Column with `~astropy.coordinates.SkyCoord` sky coordinates of the sources.
+* ``'label'``: Column with string identifiers of the sources. If you have numerical identifiers,
+  you have to recast them as string first.
```

Comment:
`label` is optional now, right?  And won't the logic here cast a column of integers to a string, or no?

Reply:
In practice, I don't think a catalog is useful without identifiers. We should encourage users to pass them in.

Reply:
The existing "From File" logic is pretty bare-bones. All I see is:

```python
                row_info = {'Right Ascension (degrees)': row['sky_centroid'].ra.deg,
                            'Declination (degrees)': row['sky_centroid'].dec.deg,
                            'Object ID': row.get('label', 'N/A')}
```

So, no, it does not turn input int column into string.

Reply:
But I added a commit to fix that as well. Please re-re-review. Thanks!